packages/react/src/hooks/__tests__/useGetURL.test.ts

```diff
@@ -0,0 +1,90 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { waitFor } from '@testing-library/react';
+import { getUrl, GetUrlInput } from 'aws-amplify/storage';
+
+import { useGetURL } from '../useGetURL';
+
+jest.mock('aws-amplify/storage');
+
+type UseGetUrlInput = GetUrlInput & {
+  onError?: (error: Error) => void;
+};
+
+describe('useGetURL', () => {
+  afterEach(() => jest.clearAllMocks());
```

Comment:
Don't think `afterEach` does anything with the return value of it's callback param or that `jest.clearAllMocks` returns anything?

Reply:
What do you mean by that? This line is clears the `getUrl` mock between tests